src/styles/colors.ts

```diff
@@ -29,6 +30,7 @@ const basicDarkColors: typeof basicLightColors = {
   peach: '#FF5B00',
   liteOrange: '#242424',
   liteGreen: '#3EBD93',
+  dollarGreen: '#009393',
```

Comment:
looks like this color is not changing for `light/dark` mode and should be added to `accentColors` instead